examples/misc/lib/effective_dart/style_bad.dart

```diff
@@ -1,20 +1,6 @@
 // ignore_for_file: constant_identifier_names, non_constant_identifier_names, type_annotate_public_apis
 import 'dart:math';
 
-int HTTPConnection, UiHandler, IoStream, HTTPRequest, ID, Db;
-List list = <int>[
-  // #docregion acronyms-and-abbreviations
-  HTTPConnection,
-  UiHandler,
-  IoStream,
-  HTTPRequest,
-  ID,
-  Db,
-  // #enddocregion acronyms-and-abbreviations
-];
```

Comment:
In retrospect, it is a bit silly to run a list of arbitrary names through the analyzer. So I'm dropping this from the example code.